EDDI/App.xaml.cs

```diff
@@ -32,6 +33,14 @@ public static void Main()
             StartRollbar(); // do immediately to initialize error reporting
             ApplyAnyOverrideCulture(); // this must be done before any UI is generated
 
+            // Start by fetching information from the update server, and handling appropriately
```

Comment:
Not sure that moving the test to here will also work for VA users: needs test

Reply:
Works in VA. :-)